hermes_cms/hermes_cms/app.py

```diff
@@ -3,11 +3,17 @@
 
 from flask import Flask, current_app
 from hermes_cms.core.registry import Registry
+from hermes_cms.core.log import setup_logging
 from sqlobject import sqlhub, connectionForURI
 
+setup_logging()
+
 
 def db_connect():
-    sqlhub.threadConnection = connectionForURI(current_app.config.get('DATABASE'))
+    database_url = current_app.config.get('DATABASE')
```

Comment:
This is currently the wrong format, and not as discussed in the developer docs. Should be ok for now though.